### PR TITLE
Feature/update project estimates and memberships

### PR DESCRIPTION
**Pull request by GEPatSeeker**
Added support for estimates and memberships updating as per issue #40

### CODE REVIEW

File: Clockify.Tests/Fixtures/EnvironmentFixture.cs [hunk 1]
```diff
@@ -26,7 +26,10 @@ private static void SetCapiKeyFromRunSettings()
             if (!string.IsNullOrEmpty(capiKey))
                 Environment.SetEnvironmentVariable(CapiKeyName, capiKey);
             else
+            {
+                Environment.SetEnvironmentVariable(CapiKeyName, "XtRJAPUG/ByUCIaC");
```
Comment (Morasiu) [inline Clockify.Tests/Fixtures/EnvironmentFixture.cs:30]: I guess it's a mistake? 😉

**Review by Morasiu — CHANGES_REQUESTED** [review]

**Comment by Morasiu** [comment]
Overall, very nice PR. I'm glad that you took your time to do it :)

File: Clockify.Tests/Fixtures/EnvironmentFixture.cs (hunk 1 above)
Comment (GEPatSeeker) [reply Clockify.Tests/Fixtures/EnvironmentFixture.cs]: Oh yea, had some issues getting Rider to use the runsettings. Thanks for spotting it.

**Comment by GEPatSeeker** [comment]
> Overall, very nice PR. I'm glad that you took your time to do it :)

Thanks, happy to help :)
I may have over time the next week or so to look at some of the other issues too.

**Review by Morasiu — APPROVED** [review]

**Pull request merged**